test/core/store/stores/test_s3.py

```diff
@@ -62,6 +66,29 @@ def test_get_open_data_params_schema(self):
         )
         self.assertEqual(set(), schema.required)
 
+    def test_get_search_params_schema(self):
+        schema = self.store.get_search_params_schema()
+        self.assertIsInstance(schema, JsonObjectSchema)
+        self.assertEqual({}, schema.properties)
+        self.assertEqual(False, schema.additional_properties)
+
+        schema = self.store.get_search_params_schema(type_specifier='geodataframe')
+        self.assertIsInstance(schema, JsonObjectSchema)
+        self.assertEqual({}, schema.properties)
+        self.assertEqual(False, schema.additional_properties)
+
+    # def test_search_data(self):
```

Comment:
If you commit code in comments, please add why this is in a comment.

EDIT: Maybe add a test like test_write_and_read_delete? Or create a new test class for the two methods?

Reply:
> If you commit code in comments, please add why this is in a comment.

I'll do.

> EDIT: Maybe add a test like test_write_and_read_delete? Or create a new test class for the two methods?

In general I agree. But I don't have the time now. It is an implementation helper and as such its code is covered multiple times by mem, dir, and s3 store tests.